common/djangoapps/track/backends/logger.py

```diff
@@ -41,6 +41,7 @@ def send(self, event):
                 "UnicodeDecodeError Event_type: %r, Event_source: %r, Page: %r, Referer: %r",
                 event.get('event_type'), event.get('event_source'), event.get('page'), event.get('referer')
             )
+            raise UnicodeDecodeError
```

Comment:
You just want to raise the original exception, so instead of `raise UnicodeDecodeError`, just write `raise`